experimental/generation/generator/packages/library/templates/standard/en-us/datetimeV2Entity.en-us.lg.lg

```diff
@@ -1,7 +1,7 @@
 [standard.en-us.lg](standard.en-us.lg)
 
 # filename
-- ${entityLGFilename()}
+- ${generationDir()}${locale}/form/${prefix}-datetimeEntity.${locale}.lg
```

Comment:
>${generationDir()}${locale}/form/${prefix}-datetimeEntity.${locale}.lg [](start = 2, length = 70)

Should probably turn this into a single parameterized function in generation.  I try not to put boiler plate in every template file.  This makes it easier to change.

Reply:
You mean that putting ${generationDir()}${locale}/form/${prefix}-datetimeEntity.${locale}.lg into standard.en-us.lg file ?